apps/notifier/src/pull-listener.ts

```diff
@@ -15,14 +21,81 @@ export default async function startPullListener(): Promise<void> {
     const subscription = await pubsubClient.getSubscription(pubsubSettings.topicName, pubsubSettings.subscriptionName)
     logger.info('listening messages from topics')
 
+    sgMail.setApiKey(SENDGRID_APY_KEY);
+
     subscription.on('message', async (message: Message) => {
       const { attributes } = message
+      const { topic } = attributes;
+      const dataJson = JSON.parse(message.data.toString());
+
+      const pathToAttachment = `${__dirname}/src/assets/views/hola.html`;
```

Comment:
That should be in another file as I explained. A good idea would be to have an object with the different types of messages that we'll sent so that we can then manage each one properly.